### DISCUSSION TITLE
Is it possible to propose/suggest new achievements?

### DISCUSSION THREAD

**Discussion by litetex** [community/community#29297] [Other Feature Feedback, Questions, & Ideas]
Is it possible to propose/suggest new achievements somewhere?

I had some interesting thoughts in last weeks but I was not able to find an documented official platform to share them...

**Comment by radar-blip** [comment]
You can start a new discussion and add '[Feature Request] Achievements'  in the title.
Then just add your description. Other members and possibly staff will see it.

**Reply by Wep-wep** [reply]
> Puede iniciar una nueva discusión y agregar '[Solicitud de funciones] Logros' en el título. Luego solo agrega tu descripción. Otros miembros y posiblemente el personal lo verán.

**Comment by ma7555** [comment]
Can you share some of your thoughts? maybe if they have some acceptance they could be added.

**Comment by blckclov3r** [comment]
Yes, If the staff noticed it

**Comment by mndree** [comment]
yess sir you can, you just have to post a discussion in this profile category and wait for the staff to notice your request

pls set my reply as the answer pls hahahaha @litetex
